### DISCUSSION TITLE
Can I use plugins with function components?

### DISCUSSION THREAD

**Discussion by erganmedia** [naver/egjs-flicking#697] [Q&A]
Hi,
I just wanted to know if it is possible to use Flicking plugins with function components. All examples are using class components.

Thank you in advance

**Comment by malangfox** [comment]
Hello @erganmedia.

You can use Flicking and plugins in function components as well. There is an example react codes in the [plugin demo](https://naver.github.io/egjs-flicking/Plugins).
If you encounter any issues during implementation, please leave a comment.
